src/pages/Survey/Survey.tsx

```diff
@@ -15,13 +26,17 @@ const SURVEY_STATE_KEY = {
   HEIGHT: 'height',
   WEIGHT: 'weight',
   SPLIT: 'split',
+  AUDIOCOACH: 'audioCoach',
```

Comment:
보통 스네이크 케이스를 사용하게 되면 단어 사이에 언더바`_`를 붙혀주는걸로 알아요 :)
```suggestion
  AUDIO_COACH: 'audioCoach',
```